sources/prompt_forwarding/remote

```diff
@@ -1,131 +1,135 @@
-
 # List of OS names and their icons
-declare -A GBT__SYMBOLS
-GBT__SYMBOLS[amzn]='\001\e[38;5;208m\002'
-GBT__SYMBOLS[arch]='\001\e[38;5;25m\002'
-GBT__SYMBOLS[archarm]='\001\e[38;5;125m\002'
-GBT__SYMBOLS[centos]='\001\e[38;5;27m\002'
-GBT__SYMBOLS[coreos]='\001\e[38;5;15m\002'
-GBT__SYMBOLS[darwin]='\001\e[38;5;15m\002'
-GBT__SYMBOLS[debian]='\001\e[38;5;88m\002'
-GBT__SYMBOLS[docker]='\001\e[38;5;26m\002'
-GBT__SYMBOLS[elementary]='\001\e[38;5;33m\002'
-GBT__SYMBOLS[fedora]='\001\e[38;5;32m\002'
-GBT__SYMBOLS[freebsd]='\001\e[38;5;1m\002'
-GBT__SYMBOLS[gentoo]='\001\e[38;5;62m\002'
-GBT__SYMBOLS[linux]='\001\e[38;5;15m\002'
-GBT__SYMBOLS[linuxmint]='\001\e[38;5;47m\002'
-GBT__SYMBOLS[mageia]='\001\e[38;5;24m\002'
-GBT__SYMBOLS[mandriva]='\001\e[38;5;208m\002'
-GBT__SYMBOLS[opensuse]='\001\e[38;5;113m\002'
-GBT__SYMBOLS[raspbian]='\001\e[38;5;125m\002'
-GBT__SYMBOLS[redhat]='\001\e[38;5;1m\002'
-GBT__SYMBOLS[sabayon]='\001\e[38;5;15m\002'
-GBT__SYMBOLS[slackware]='\001\e[38;5;15m\002'
-GBT__SYMBOLS[ubuntu]='\001\e[38;5;166m\002'
+GBT__SYMBOLS_AMZN='\001\e[38;5;208m\002'
```

Comment:
@jtyr what are these `\001` and `\002` escape characters? They seem to cause an issue in `sh` but removing them seems to be fine for `bash`